pages/agenda/2018.tsx

```diff
@@ -0,0 +1,280 @@
+              </tr>
+
+              <tr className="breadth-row">
+                <td className="time">14:25</td>
+                <td colSpan={4} className="breadth">
+                  Changeover
+                </td>
+              </tr>
+
+              <tr>
+                <td className="time">14:35</td>
+                <SessionCell sessionId="f7fc010d-8c47-4b86-b1e9-1221b63e0281" />
+                <SessionCell sessionId="2234f8f6-6e13-4998-ba37-baf53ae44d9d" />
+                <SessionCell sessionId="4f463f9b-bf28-446a-9558-c6ac59697cc9" />
+                <SessionCell sessionId="fd830fb5-8e7b-4527-bc79-f7ddf693232f" />
+              </tr>
+
+              <tr className="breadth-row">
+                <td className="time">15:20</td>
+                <td colSpan={4} className="breadth">
+                  Afternoon tea
+                  <br />
+                  <small className="room">Upstairs foyer</small>
+                </td>
+              </tr>
+
+              <tr className="breadth-row">
+                <td className="time">15:50</td>
+                <td colSpan={4} className="breadth">
+                  Prize Draw
+                  <br />
+                  <small className="room">VGW Ballroom 2</small>
+                </td>
+              </tr>
+
+              <tr className="breadth-row">
+                <td className="time">16:20</td>
+                <SessionCell
+                  isLocknote={true}
+                  sessionId="264b7669-8127-41a3-9f6b-87511a879cf1"
+                  sponsorName="YOW! Perth"
+                  room="VGW Ballroom 2"
+                />
+              </tr>
+
+              <tr className="breadth-row">
+                <td className="time">17:05</td>
+                <td colSpan={4} className="breadth">
+                  <strong>Thank yous and wrap up</strong>
+                  <br />
+                  <small className="room">VGW Ballroom 2</small>
+                </td>
+              </tr>
+
+              <tr className="breadth-row">
+                <td className="time">17:10</td>
+                <td colSpan={4} className="breadth">
+                  <strong>Afterparty</strong>
+                  <br />
+                  <small className="room">Upstairs foyer</small>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <h2>Handbook</h2>
+        <p>
+          <a className="btn btn-pdf" href={From2018.HandbookUrl}>
+            Download 2018 handbook (PDF)
+          </a>
+        </p>
+        <h2>Media</h2>
+
+        <div className="text-center">
+          <ResponsiveVideo src={From2018.YouTubeKeynoteEmbedUrl} />
+          <ResponsiveVideo src={From2018.YouTubeLocknoteEmbedUrl} />
+        </div>
+        <br />
+        <div>
+          <ResponsiveVideo src={From2018.YouTubeLunchnoteEmbedUrl} />
+        </div>
+        <p>
+          <a href={From2018.YouTubePlaylistUrl} target="_blank">
```

Comment:
Is it worth creating a safe link component that adds rel="noopener" to blank target links?
https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
Agreed, feel free to correct / add to backlog